samples/DispatcherSample.Web/Startup.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Routing.Matchers;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DispatcherSample.Web
+{
+    public class Startup
+    {
+        private static readonly byte[] _helloWorldPayload = Encoding.UTF8.GetBytes("Hello, World!");
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddRouting();
```

Comment:
It needs to for now since it will use some services that are included by routing. I think whether or not we have `AddRouting` + `AddXyz` will depend on the final naming and how much they share when we're further along. 

I'd lean towards having a single method... which would be `AddRouting` - the cost of 'extra' services you don't use is basically nothing.